areas-of-d-usage.dd

```diff
@@ -0,0 +1,189 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(DIVC page-contents quickindex,
+    $(DIVC page-contents-header,
+        $(H3 Contents)
+    )
+    $(OL
+        $(LI $(RELATIVE_LINK2 system_programming, System programming)) $(OL
+            $(LI $(RELATIVE_LINK2 operating_systems, Operating systems))
+            $(LI $(RELATIVE_LINK2 compilers, Compilers))
+            $(LI $(RELATIVE_LINK2 embedded, Embedded applications))
+        )
+        $(LI $(RELATIVE_LINK2 industry, Industry)) $(OL
+            $(LI $(RELATIVE_LINK2 games, Games))
+            $(LI $(RELATIVE_LINK2 web, Web applications))
+            $(LI $(RELATIVE_LINK2 gui, GUI applications))
+        )
+        $(LI $(RELATIVE_LINK2 academia, Academia)) $(OL
+            $(LI $(RELATIVE_LINK2 numerical, Numerical computing))
+            $(LI $(RELATIVE_LINK2 gpu, GPU programming))
+            $(LI $(RELATIVE_LINK2 data_science, Data science))
+            $(LI $(RELATIVE_LINK2 teaching, Teaching))
+        )
+    )
+)
+
+This list highlights a selection of great applications written in D
+from various areas. $(BR)
```

Comment:
I dislike these `$(BR)`s throughout. I'd rather see paragraphs wrapped in `$(P ...)`. In cases where that adds to much space, I'd say just join the lines, as there is no need for separation then.